.cirrus.yml

```diff
@@ -505,3 +505,30 @@ container_image_manifest_docker_builder:
   depends_on:
     - arm64_container_image
     - amd64_container_image
+
+cluster_testing_docker_builder:
+  cpu: *CPUS
+  memory: *MEMORY
+  only_if: $CIRRUS_REPO_FULL_NAME == 'zeek/zeek'
```

Comment:
Should this be limited to not run on every PR?

Reply:
I'm fine either way - the GitHub action currently runs on PRs and those tests are not otherwise covered by the existing tasks.

Reply:
I'd leave it for now.